docs/PLUGINS.md

```diff
@@ -212,10 +212,9 @@ Add the Configuration as Code plugin as a test dependency in your pom.xml:
     <scope>test</scope>
 </dependency>
 <dependency>
-    <groupId>io.jenkins</groupId>
-    <artifactId>configuration-as-code</artifactId>
+    <groupId>io.jenkins.configuration-as-code</groupId>
```

Comment:
This needs to be documented in the release notes so when dependabot does the PR it very transparent that this is the change we did (since we have removed the test jar under classifier `tests`)